### DISCUSSION TITLE
Will Rasberry pi5 work with current file?

### DISCUSSION THREAD

**Discussion by BroRidersMTL** [tesla-android/issue-tracker#311] [General]
Hi, wondering if latest teslaandroid will work with pi5? if not will you be releasing an update for pi5?

**Comment by mikegapinski** [comment]
No, the Pi 5 does not have hardware video/image encoders that are essential for TA. It will be slower without them, it makes no sense

Wysyłane z aplikacji Outlook dla systemu iOS<https://aka.ms/o0ukef>
________________________________
Od: BroRidersMTL ***@***.***>
Wysłane: Saturday, February 24, 2024 9:38:48 PM
Do: tesla-android/issue-tracker ***@***.***>
DW: Subscribed ***@***.***>
Temat: [Tesla Android] Will Rasberry pi5 work with current file? (Discussion #311)


Hi, wondering if latest teslaandroid will work with pi5? if not will you be releasing an update for pi5?

**Reply by BroRidersMTL** [reply]
Thats cray cray thanks for heads up....do you have an eta for CM5?

**Comment by mikegapinski** [comment]
CM5 is not out yet but it’s the same chip as on the Pi 5, so same encoding issues

Wysyłane z aplikacji Outlook dla systemu iOS<https://aka.ms/o0ukef>
________________________________
Od: BroRidersMTL ***@***.***>
Wysłane: Sunday, February 25, 2024 12:14:13 AM
Do: tesla-android/issue-tracker ***@***.***>
DW: Michał Gapiński ***@***.***>; Comment ***@***.***>
Temat: Re: [tesla-android/issue-tracker] Will Rasberry pi5 work with current file? (Discussion #311)


Thats cray cray thanks for heads up....do you have an eta for CM5?
